acceptance/acceptance_test.go

```diff
@@ -56,6 +56,7 @@ const (
 	EntryPointScript = "script"
 	CleanupScript    = "script.cleanup"
 	PrepareScript    = "script.prepare"
+	ReplsFile        = "repls.json"
```

Comment:
Could you add a comment what's this file used for? From first glance it's not clear if it's an input for replacement or some sort of output

Reply:
Added a comment.